flathub.json

```diff
@@ -1,3 +1,3 @@
 {
-  "only-arches": ["x86_64"]
+  "only-arches": ["x86_64", "aarch64"]
```

Comment:
you can remove this file, it's not needed. these are the only two arches supported.